src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -1020,8 +1025,8 @@ class Simpletask : ThemedNoActionBarActivity() {
             mFilter!!.saveInIntent(intent)
             setIntent(intent)
             mFilter!!.saveInPrefs(Config.prefs)
-            m_adapter!!.setFilteredTasks()
             closeDrawer(NAV_DRAWER)
+            closeSelectionMode()
```

Comment:
`closeSelectionMode` calls `m_adapter!!.setFilteredTasks()`, so we don't need to do it twice.